src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -39,6 +49,13 @@ public static long sumAll(List<Integer> numbers) {
     }
 
     public static long sumOverThreeAndDouble(List<Integer> numbers) {
-        return 0;
+        if(numbers == null){
+            return 0;
+        }
+
+        return numbers.stream()
+                .filter(number -> number > 3)
+                .map(number -> number * 2)
+                .reduce(0, (x, y) -> x + y);
```

Comment:
```suggestion
                .reduce(0, Integer::sum);
```
x + y가 직관적이긴 하지만 공유 차원에서 남겨요